app/access_grant_funding/templates/access_grant_funding/accessibility-statement.html

```diff
@@ -0,0 +1,32 @@
+{% extends "access_grant_funding/base.html" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">Accessibility statement for Access grant funding</h1>
+
+      <p class="govuk-body">
+        This accessibility statement applies to
+        <a class="govuk-link govuk-link--no-visited-state" href="https://funding.communities.gov.uk/access">Access grant funding</a>.
+      </p>
+
+      <h2 class="govuk-heading-m">Technical information about this website's accessibility</h2>
+      <p class="govuk-body">The MHCLG is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
+
+      <h3 class="govuk-heading-s">Compliance status</h3>
+      <p class="govuk-body">
+        The website has not yet been tested against the <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/TR/WCAG22/">Web Content Accessibility Guidelines (WCAG) version 2.2&nbsp;AA standard</a>.
+      </p>
+      <p class="govuk-body">
+        (c) This website is not compliant with the <a class="govuk-link govuk-link--no-visited-state" href="https://www.w3.org/TR/WCAG22/">Web Content Accessibility Guidelines (WCAG) version 2.2&nbsp;AA standard</a>. We have not yet had an
```

Comment:
is (c) from somewhere else where there was a longer list and might not be needed?

This is as-is in the ticket so approving either way